tests/Validators/StringValidatorTest.php

```diff
@@ -0,0 +1,33 @@
+<?php
+
+namespace Tests\Validators;
+
+use App\Validators\StringValidator;
+use Tests\TestCase;
+
+class StringValidatorTest extends TestCase
+{
+    public function testValidateExistsAndLengthSuccess()
+    {
+        $maxCharacterLength = 30;
+        $name = 'Loads of stocks';
+        $result = StringValidator::validateExistsAndLength($name, $maxCharacterLength);
+        $this->assertEquals('Loads of stocks', $result);
+    }
+
+    public function testValidateExistsAndLengthFailure()
+    {
+        $maxCharacterLength = 30;
+        $name = '';
+        $this->expectException(\Exception::class);  
```

Comment:
This should be more specific for our case. This just expects any exception but we are expecting our specific custom exception.